_data/people.yaml

```
@@ -357,6 +456,11 @@ emmyft:
     email: e.tsang@elifesciences.org
     website: https://elifesci.org/labs
     orcid: 0000-0002-9248-1280
+    expertise:
+        - Community strategies
+        - Publishing and scholarly communication
+life-science-expertise: |    
```

```suggestion
    life-science-expertise: |      
```

I think this fixes the syntax error on my machine